flatfs.go

```diff
@@ -33,6 +37,9 @@ type Datastore struct {
 
 	// sychronize all writes and directory changes for added safety
 	sync bool
+
+	diskUsage   uint64
+	diskUsageCh chan int64
```

Comment:
Why not just use `atomic.AddInt64` (and store it as an int64)?

Reply:
yes, indeed..